app/main/routes.py

```diff
@@ -92,106 +96,232 @@ def accessibility():
+        # all access user (superuser)
+        for body in Body.query.all():
+            transferring_bodies.append(body.Name)
 
-    if transferring_body_id:
-        browse_type = "transferring_body"
-        browse_parameters["transferring_body_id"] = transferring_body_id
-        breadcrumb_values = {
-            0: {"transferring_body": Body.query.get(transferring_body_id).Name}
-        }
         filters = build_filters(request.args)
         sorting_orders = build_sorting_orders(request.args)
 
-    elif series_id:
-        browse_type = "series"
-        browse_parameters["series_id"] = series_id
-        series = Series.query.get(series_id)
-        body = series.body
-        breadcrumb_values = {
-            0: {"transferring_body_id": body.BodyId},
-            1: {"transferring_body": body.Name},
-            2: {"series": series.Name},
-        }
+        query = build_browse_all_query(
+            filters=filters,
+            sorting_orders=sorting_orders,
+        )
 
-        filters = build_filters(request.args)
-        sorting_orders = build_sorting_orders(request.args)
+        browse_results = query.paginate(page=page, per_page=per_page)
 
-    elif consignment_id:
-        browse_type = "consignment"
-        browse_parameters["consignment_id"] = consignment_id
-
-        consignment = Consignment.query.get(consignment_id)
-        body = consignment.series.body
-        series = consignment.series
-        breadcrumb_values = {
-            0: {"transferring_body_id": body.BodyId},
-            1: {"transferring_body": body.Name},
-            2: {"series_id": series.SeriesId},
-            3: {"series": series.Name},
-            4: {"consignment_reference": consignment.ConsignmentReference},
-        }
-
-        filters = build_browse_consignment_filters(request.args)
-        sorting_orders = build_sorting_orders(request.args)
+        total_records = db.session.query(
+            func.sum(query.subquery().c.records_held)
+        ).scalar()
+        if total_records:
+            num_records_found = total_records
+        else:
+            num_records_found = 0
+
+        return render_template(
+            "browse.html",
+            form=form,
+            current_page=page,
+            browse_type="browse",
+            results=browse_results,
+            transferring_bodies=transferring_bodies,
+            filters=filters,
+            sorting_orders=sorting_orders,
+            num_records_found=num_records_found,
+            dict_args={
+                k: v for k, v in request.args.items() if k not in "page"
+            },
+        )
+
+
+@bp.route("/browse/transferring_body/<uuid:_id>", methods=["GET"])
+@access_token_sign_in_required
+def browse_transferring_body(_id: uuid.UUID):
+    """
+    Render the browse transferring body view page.
+
+    This function retrieves search results for a specific
+    record(s) based on the 'transferring_body_id' provided in the query parameters, and renders
```

Comment:
```suggestion
    record(s) based on the transferring_body 'id' provided
```

as its not from the query params anymore

Reply:
updated comment for transferring body, series and consignment routes as requested

Reply:
cheers